packages/models/proto/v2/eservice/events.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package eservice.v2;
+
+import "v2/eservice/eservice.proto";
+
+message EServiceAddedV2 {
+  EServiceV2 eservice = 1;
+}
+
+message DraftEServiceUpdatedV2 {
+  EServiceV2 eservice = 1;
+}
+
+message EServiceDeletedV2 {
+  string eserviceId = 1;
+  EServiceV2 eservice = 2;
+}
+
+message EServiceClonedV2 {
+  EServiceV2 sourceEservice = 1;
+  string sourceDescriptorId = 2;
+  EServiceV2 clonedEservice = 3;
+}
+
+message EServiceDescriptorAddedV2 {
+  string descriptorId = 1;
+  EServiceV2 eservice = 2;
+}
+
+message EServiceDraftDescriptorUpdatedV2 {
+  string descriptorId = 1;
+  EServiceV2 eservice = 2;
+}
+
+message EServiceDescriptorActivatedV2 {
+  string descriptorId = 1;
+  EServiceV2 eservice = 2;
+}
+
+message EServiceDescriptorArchivedV2 {
+  string descriptorId = 1;
+  EServiceV2 eservice = 2;
+}
+
+message EServiceDescriptorPublishedV2 {
+  string descriptorId = 1;
+  optional string deprecatedDescriptorId = 2;
```

Comment:
This name should be changed or we should find another wait to add this information to this event

Reply:
we can remove this field, in case of need the deprecated descriptor can be deduced from the event meaning

Reply:
@galales I am not sure about your point. Remember the other descriptor will have a different state based on some [additional logic](https://github.com/pagopa/interop-be-monorepo/pull/208/files#diff-77a5bad1f3b877e7bfc968a9b423c16649fca358d54286d743fe34fa555564ceR1072-R1102).

Reply:
@taglioni-r yes, but 
- the other descriptor state will be changed upstream by the process
- consumers that use this event will know that on publication the second-to-last descriptor has been deprecated or archived

Given these points there is no need to add the reference of the deprecated/archived descriptor.
This also to avoid hardwiring several business logics to the event schema (the event name should be enough)